preview/views/VerticalLayout.hpp

```diff
@@ -50,9 +50,9 @@ namespace services
             using HorizontalLayouter::HorizontalLayouter;
 
         protected:
-            virtual infra::Point AdjustOrientation(infra::Point point) const;
```

Comment:
I thought virtual and override should be used here? Per the coding standard in EmbeddedInfraLib: "Rule 52. When overriding virtual functions in a derived class, the keywords virtual and override shall be used to explicitly declare the function to override a base function."